lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb

```diff
@@ -26,7 +31,10 @@ def self.xcodebuild(options)
       cmd << "2>&1"
       cmd = cmd.join(" ")
 
-      Pod::UI.puts_indented "$ #{cmd}"
+      if !PodPrebuild.config.silent_build?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/b144f30fb41bd34e49fceb072710e5f2c3a3e710/lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb#L34"></span>Style/IfUnlessModifier: Favor modifier <code>if</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>